csrankings-m.csv

```diff
@@ -542,7 +542,7 @@ Maria Luisa Gil Gomez,Polytechnic University of Catalonia,http://directori.upc.e
 Maria Luísa Torres Ribeiro Marques da Silva Coheur,Universidade de Lisboa,https://www.l2f.inesc-id.pt/wiki/index.php/Lu%C3%ADsa_Coheur,dJ5sl_QAAAAJ
 María Martínez Pérez,University of A Coruña,https://pdi.udc.es/en/File/Pdi/AW2EH,NOSCHOLARPAGE
 Maria Montanucci,DTU,https://www.compute.dtu.dk/english,NOSCHOLARPAGE
-Maria Paasivaara,IT University of Copenhagen,https://pure.itu.dk/portal/en/persons/maria-paasivaara(f0564851-073d-4f4e-8dc5-c2c179799843).html,T4iNd0gAAAAJ
+Maria Paasivaara,Aalto University,https://people.aalto.fi/maria.paasivaara,T4iNd0gAAAAJ

```

Comment:
Listed as an adjunct rather than a full faculty member, so should be removed.

Reply:
Right. This is incorrect, Aalto was her former co-affiliation. She moved to "LUT University" (https://research.lut.fi/converis/portal/detail/Person/13753780), not Aalto. Sorry. 

I can't find LUT University in CSRankings. What to do in this case?

Reply:
I removed the entry now and leave it to someone who knows CS at LUT to add faculty to CSRankings.